### PR TITLE
Add cookiewow

### PR DESCRIPTION
**Pull request by whs**
Cookiewow.com is a Thai cookie consent provider, used by some Thai sites like thestandard.co, techsauce.co .

### CODE REVIEW

File: rules/cookiewow.json [hunk 1]
```diff
@@ -0,0 +1,234 @@
+{
+	"$schema": "../rules.schema.json",
```
Comment (whs) [inline rules/cookiewow.json:2]: I actually hand crafted this, with VS Code and JSON schema. VSCode use $schema to point to the schema definition. I'd be open to upload this in a separate PR.

Comment (dkgof) [reply rules/cookiewow.json:2]: If you made a schema for the rules, that would be pretty cool to have. Even just a start that we can build upon.
